### PR TITLE
make invisible container not mess with mouse events

### PR DESCRIPTION
**Pull request by hurradieweltgehtunter**
Invisible notifications container blocked clicking on call-to-actions beneath

### CODE REVIEW

**Comment by phil-davis** [comment]
CTA = https://en.wikipedia.org/wiki/Call_to_action ?

**Comment by hurradieweltgehtunter** [comment]
Yep. Edited my comment. Thought it was a common phrase

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/26913/3/2
`file changelog/unreleased/37941: entry title does not have a prefix, example: Bugfix: restore old behavior`

File: changelog/unreleased/37941 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Bugfix: In addition to 37875 this fixes the invisible notification container blocking mouse events on cta's beneath it
```
Comment (phil-davis) [inline changelog/unreleased/37941:1]:
```suggestion
Bugfix: stop UI notifications from blocking mouse events on items beneath them
```

https://drone.owncloud.com/owncloud/core/26931/3/2
`file changelog/unreleased/37941: title is too long`

**Comment by hurradieweltgehtunter** [comment]
fixed the changelog but now ApiShareManagement Acceptance Tests fail?

**Comment by phil-davis** [comment]
Some test suites have a been split into 2 suites recently in master.
There is a catch-22 for that sort of thing - the `.drone.star` in this branch has an old list of test suites, but the code in master has a different list.

Rebase the PR to latest master.

**Comment by hurradieweltgehtunter** [comment]
Yeah I squashed the changelog commits and rebased the branch

**Comment by phil-davis** [comment]
You are so lucky!!!
https://drone.owncloud.com/owncloud/core/26933/47/6
The pipeline is stuck in the `yarn-install` step.
I cancelled and restarted drone CI.

**Comment by hurradieweltgehtunter** [comment]
xD just saw it. Thank you

**Comment by hurradieweltgehtunter** [comment]
Codecov stuck? Started 4 days ago

**Comment by phil-davis** [comment]
codecov is always stuck. We are moving to SonarCloud. In the interim time, codecov results are not "Required" for merge.

**Review by phil-davis — APPROVED** [review]
Works - with this PR code, I can
1) have a long list of files in the UI
2) have a notification displayed at the bottom of the browser, the notification icon and text is over a file row
3) click on the file name or sharing icon or "..." actions menu and they react

Without this code, item (3) is not possible

**Pull request merged**